docs/examples/Reinforcement_Learning_Sensor_Manager.py

```diff
@@ -0,0 +1,1002 @@
+    truths.append(truth)
+
+    # alternate directions when initiating tracks
+    xdirection *= -1
+    if j % 2 == 0:
+        ydirection *= -1
+
+# %%
+# Plot the ground truths. This is done using the :class:`~.Plotterly` class from Stone Soup.
+
+from stonesoup.plotter import AnimatedPlotterly
+
+plotter = AnimatedPlotterly(timesteps, tail_length=1)
+plotter.plot_ground_truths(truths, [0, 2])
+plotter.fig
+
+# %%
+# Create sensors
+# --------------
+# Create a sensor for each sensor management algorithm. This tutorial uses the
+# :class:`~.RadarRotatingBearingRange` sensor. This sensor is an :class:`~.Actionable` so
+# is capable of returning the actions it can possibly
+# take at a given time step and can also be given an action to take before taking
+# measurements.
+# See the Creating an Actionable Sensor Example for a more detailed explanation of actionable sensors.
+# 
+# The :class:`~.RadarRotatingBearingRange` has a dwell centre which is an :class:`~.ActionableProperty`
+# so in this case the action is changing the dwell centre to point in a specific direction.
+# 
+
+from stonesoup.types.state import StateVector
+from stonesoup.sensor.radar.radar import RadarRotatingBearingRange
+
+sensorA = RadarRotatingBearingRange(
+    position_mapping=(0, 2),
+    noise_covar=np.array([[np.radians(0.5) ** 2, 0],
+                          [0, 1 ** 2]]),
+    ndim_state=4,
+    position=np.array([[10], [0]]),
+    rpm=60,
+    fov_angle=np.radians(45),
+    dwell_centre=StateVector([0.0]),
+    max_range=np.inf
+)
+sensorA.timestamp = start_time
+
+sensorB = RadarRotatingBearingRange(
+    position_mapping=(0, 2),
+    noise_covar=np.array([[np.radians(0.5) ** 2, 0],
+                          [0, 1 ** 2]]),
+    ndim_state=4,
+    position=np.array([[10], [0]]),
+    rpm=60,
+    fov_angle=np.radians(45),
+    dwell_centre=StateVector([0.0]),
+    max_range=np.inf
+)
+sensorB.timestamp = start_time
+
+# %%
+# Create the Kalman predictor and updater
+# ---------------------------------------
+# Construct a predictor and updater using the :class:`~.KalmanPredictor` and :class:`~.ExtendedKalmanUpdater`
+# components from Stone Soup. The :class:`~.ExtendedKalmanUpdater` is used because it can be used for both linear
+# and nonlinear measurement models.
+# 
+
+from stonesoup.predictor.kalman import KalmanPredictor
+predictor = KalmanPredictor(transition_model)
+
+from stonesoup.updater.kalman import ExtendedKalmanUpdater
+updater = ExtendedKalmanUpdater(measurement_model=None)
+# measurement model is added to detections by the sensor
+
+# %%
+# Run the Kalman filters
+# ----------------------
+# First create `ntruths` priors which estimate the targets’ initial states, one for each target. In this example
+# each prior is offset by 0.5 in the y direction meaning the position of the track is initially not very accurate. The
+# velocity is also systematically offset by +0.5 in both the x and y directions.
```

Comment:
Actually the offsets look like nothing in x, +0.1 in y, and +0.2 in xdot and ydot.